b2share/factory.py

```diff
@@ -116,3 +116,7 @@ def serve_records(path):
     @app_ui.route('/search/<path:path>')
     def serve_search(path):
         return app_ui.send_static_file('index.html')
+
+    @app_ui.route('/languages.json', defaults={'path': ''})
```

Comment:
I would add here a /static endpoint for serving static files, and languages.json would be only one of them (disciplines would be another). like this:
````
@app_ui.route('/static/<path:filename>')
    def serve_static_resources(filename):
        return app_ui.send_from_directory('static', filename)
````